servers.json

```diff
@@ -1899,5 +1899,15 @@
         "primaryAddress": "play.sunrisecity.uk",
         "minecraftVersions": ["1.17", "1.18"],
         "primaryMinecraftVersion": "1.17"
+    },
+    {
+        "id": "minedestiny",
+        "name": "MineDestiny",
+        "addresses": [
+            "play.minedestiny.hu"
```

Comment:
```suggestion
            "minedestiny.hu"
```

Reply:
Addresses within the 'addresses' array are not used for player connections.
They are used to match any possible subdomains to the same domain and therefor do not need to contain any subdomains.